src/openassetio-python/package/setup.py

```diff
@@ -39,10 +40,22 @@ def build_extension(self, _ext: Extension):
 
         @param _ext: Extension to build (we only have one, so ignored).
         """
+        cmake_project_path = pathlib.Path("../../..")
+
+        if not os.path.isfile(cmake_project_path / "CMakeLists.txt"):
+            # Unfortunately there doesn't seem to be a way to detect the
+            # version of pip that launched this process. So assume if
+            # CMakeLists.txt is missing then its probably because the
+            # wrong version of pip was used.
+            raise FileNotFoundError(
+                "CMakeLists.txt not found. OpenAssetIO wheel building requires an in-tree build"
+                " with setup.py at the cwd. If using pip, ensure pip>=21.3."
```

Comment:
[Consider] I'm guessing `cwd` is current working directory? Took me a little while to puzzle it out, worth expanding the jargon away I'd say.

Reply:
:grin: Fair enough, expanded in cad3ac91